apps/parsley/jest.config.js

```diff
@@ -17,7 +17,6 @@ module.exports = {
   snapshotSerializers: ["@emotion/jest/serializer"],
   testEnvironment: "jsdom",
   testMatch: ["<rootDir>/{src,scripts}/**/*.{spec,test}.{js,jsx,ts,tsx}"],
-  testRunner: "<rootDir>/node_modules/jest-circus/runner.js",
```

Comment:
This reference to `<rootDir>/node_modules` was no longer valid now that node_modules are installed in the root, but jest-circus [is the default runner](https://jestjs.io/docs/configuration#testrunner-string) so this line is superfluous anyway